metabrainz/templates/index/code-of-conduct.html

```diff
@@ -25,22 +25,22 @@ <h1 class="page-title">{{ _('Code of Conduct') }}</h1>
       agree with them.
     </li>
     <li>
-      MetaBrainz projects have global communities. Be mindful and respectful of different languages, habits and cultures.
+      MetaBrainz projects have global communities. Be mindful and respectful of different languages, habits, and cultures.
```

Comment:
We don’t generally use Oxford comma in MetaBrainz—and we also generally don’t use it. It depends on who’s written the text. E.g., @mayhem doesn’t use it, but I do. We may want to make an editorial decision whether to use it or not maybe?